.github/workflows/maven.yml

```
@@ -23,6 +23,10 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.11
+        
+  #  - name: Log into registry
```

maybe a bit nicer ;) 

```suggestion
        - name: Login to GitHub Container Registry
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: ${{ github.repository_owner }}
          password: ${{ secrets.CR_PAT }}
```